Code/src/main/java/nl/utwente/viskell/ui/components/Connection.java

```diff
@@ -226,7 +226,7 @@ private void checkErrorListener(ObservableValue<? extends Boolean> value, Boolea
     /**
      * @return Whether or not both sides of this Connection are connected to an Anchor.
      */
-    public final boolean isConnected() {
+    public final boolean isFullyConnected() {
```

Comment:
isConnected exists already as unrelated internal method in javafx.
And fullyConnected is clearer (I was also trying to track down a bug with partially connected wires staying on the uipane)